package.json

```diff
@@ -26,7 +26,7 @@
     "mocha": "2.x"
   },
   "dependencies": {
-    "iconv-lite": "0.x",
+    "iconv-lite": ">= 0.4 < 1",
```

Comment:
Why limit upper bound?

Reply:
Because it was limited before this change too (to `0.x`, effectively `< 1`). Probably good to keep it like that for compatibility reasons. Added lower bound only because it's required for `ksc5601`.